gt2_test_runner/__init__.py

```diff
@@ -546,7 +553,27 @@ def coverage_report(self, save_data=False, html_dir=None, to_stream=True):
 
             print('\nHTML coverage data is saved as file://{}/index.html'.format(html_dir),
                   file=self.stream)
+    
+    def setup_buffer(self, test):
+        sys.stdout = self.test_data[test_fqn]['standard out'] = io.StringIO()
+        sys.stderror = self.test_data[test_fqn]['standard error'] = io.StringIO()
+
+    def restore_buffer(self, test):
+        test_fqn = self._get_test_fqn(test)
 
+        output = self.test_data[test_fqn]['standard out'].getvalue()
+        error = self.test_data[test_fqn]['standard error'].getvalue()
+        
+        self.test_data[test_fqn]['standard out'].close()
+        self.test_data[test_fqn]['standard error'].close()
+        
+        sys.stdout = self._original_stdout
+        sys.stderror = self._original_stderr
```

Comment:
You don’t save these anywhere in the code, maybe you want to do so in `setup_buffer`?